playbooks/deploy.yml

```
@@ -68,13 +69,13 @@
         - "{{ lookup('template', '../openshift/sandbox-namespace.yml.j2') | from_yaml }}"
 
     - block:
-      - name: Get metadata about d.io/u-c/packit-service:{{ deployment_image }}
-        shell: skopeo inspect docker://docker.io/usercont/packit-service:{{ deployment_image }}
+      - name: Get metadata about {{ image }}:{{ deployment }}
+        shell: skopeo inspect {{ skopeo_transport }}://{{ image }}:{{ deployment }}
```

```suggestion
        shell: skopeo inspect {{ skopeo_transport }}{{ image }}:{{ deployment }}
```
where `skopeo_transport` defaults to `"docker://"`, but can be `docker-daemon:` (would be nice to comment this in `vars/template.yml`). See `IMAGE NAMES` in `man skopeo`. (I haven't invented that inconsistancy)